.eslintrc.test.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  env: {
```

Comment:
Really minor nit, perhaps can consider using double quotes here, `"env"`, so that it is consistent with the `eslintrc.js` file. Similar comment for the rest of the file :)